docs/google/calibration.md

```diff
@@ -0,0 +1,204 @@
+state after the operations (RB or XEB) have been applied.  The purity error,
+or impurity, is one minus the purity, scaled to be from zero to one.  This is
+often referred to as the normalized
+[Linear Entropy](https://en.wikipedia.org/wiki/Linear_entropy).
+after the randomized benchmark circuit has been applied.  
+
+The purity error can be interpreted as a measure of the incoherent error,
+such as those caused by stochastic processes such as relaxation.  The average
+error can be interpreted as containing both this incoherent error as well as
+the coherent error resulting from improper control or calibration of the device.
+
+Note that, due to statistical fluctuations, it is possible that the purity
+error can exceed the average error by small amounts.
+
+For more about purity benchmarking, see Section 6.3 of this
+[thesis](https://web.physics.ucsb.edu/~martinisgroup/theses/Chen2018.pdf) .
+
+## Individual Metrics
+
+Each metric can be referenced by its key in the calibration object, e.g.
+```latest_calibration['single_qubit_idle_t1_micros']```.
+
+Note that the metric names below are subject to change without notice.
+
+### P_00 readout error
+
+*   Metric key: single_qubit_readout_p00_error
+
+The p_00 is defined as the probability that the state is measured as |0⟩ after
+being prepared in the |0⟩ state.  The p_00 error is defined as one minus this
+result.  
+
+There are several sources of error in this model.  This error is primarily
+composed of error in measurement (read out) of the qubit while in the ground
+state.  However, this probability also contains the error than the qubit was not
+reset into the |0⟩ ground state properly.  This is often called the SPAM (state 
+preparation and measurement) error.
+
+### P_11 readout error
+*   Metric key: single_qubit_readout_p11_error
+
+The p_11 is defined as the probability that the state is measured as |1⟩ after
+being prepared in the |1⟩ state.  The p_11 error is defined as one minus this
+result.  
+
+This is dominated by the error in measurement (read out) of the qubit, but it
+implicitly contains several different types of error.  Also possible is that the
+excited state |1⟩ was not prepared correctly or that the state decayed before
+measurement.  This error is generally expected to be higher than the P_00 error.
+
+### Readout separation error
+*   Metric key: single_qubit_readout_separation_error
+
+When measured by the system, the |0⟩ and |1⟩ states manifest as outgoing analog
+signals.  These signals must be interpreted as being in one state or the other.
+Since these analog signals are continuous distributions, there will be some
+statistical overlap in the two distributions that would be theoretically
+impossible to distinguish.  This is classified as the separation error, and is
+calculated by fitting Gaussian distributions to the signals prepared in the
+|0⟩ state and |1⟩ state and calculating the overlap between the two
+distributions.
```

Comment:
I still think that our clients won't know how to use this. What's the goal in exposing this metric?

Reply:
This could potentially be discussed at the SPWG core meeting.  For now, we still expose this metric, so I'll leave the explanation in.  We can always delete it if we remove the metric.